oci/client/list_test.go

```diff
@@ -35,7 +35,18 @@ func Test_List(t *testing.T) {
 	ctx := context.Background()
 	c := NewClient(DefaultOptions())
 	repo := "test-list" + randStringRunes(5)
-	tags := []string{"v0.0.1", "v0.0.2", "v0.0.3", "v6.0.0", "v6.0.1", "v6.0.2", "v6.0.2-rc.1", "v6.0.2-alpha", "staging-fb3355b"}
+	appTags := []string{
+		"v0.0.1", "v0.0.2", "v0.0.3", "v6.0.0", "v6.0.1", "v6.0.2", "v6.0.2-rc.1", "v6.0.2-alpha",
+		"staging-fb3355b",
+	}
+	cosignTags := []string{
+		"sha256-e2688bb75ee43df49c9bfb2aa30dd98173649db53955e87c347024ba71bc1c80.sig",
+		"sha256-e2688bb75ee43df49c9bfb2aa30dd98173649db53955e87c347024ba71bc1c80.sbom",
+		"sha256-e2688bb75ee43df49c9bfb2aa30dd98173649db53955e87c347024ba71bc1c80.att",
+	}
+	allTags := append([]string{}, cosignTags...)
+	allTags = append(allTags, appTags...)
```

Comment:
Well, yes, but it actually adds to the underlying slice of `allTags`, which has issues, so I prefer to be explicitly safe.

Reply:
I assume you meant `allTags := append(appTags, cosignTags...)`... 
Yes, it largely does the same thing, but it will use the underlying location of `appTags`, and it's possible to end up with a combined slice in the wrong place.  I prefer to allocate a new slice explicitly.